pkg/fileutils/fileutils.go

```diff
@@ -1,33 +1,105 @@
 package fileutils
 
 import (
+	"errors"
 	"fmt"
 	"io"
 	"io/ioutil"
 	"os"
 	"path/filepath"
+	"strings"
 
 	"github.com/Sirupsen/logrus"
 )
 
+func Exclusion(pattern string) bool {
+	return pattern[0] == '!'
```

Comment:
Thanks I've trimmed the patterns in CleanPatterns now before they get passed to Exclude.